megamek/src/megamek/common/AmmoType.java

```diff
@@ -11609,97 +11604,6 @@ private static AmmoType createAR10WhiteSharkAmmo() {
                     .setReintroductionFactions(F_FS,F_LC);
                 return ammo;
             }
-
-            private static AmmoType createAR10BarracudaTAmmo() {
```

Comment:
Where did these deleted methods go? And why?

Reply:
Per the TM tables, the AR10 is not supposed to be able to use telemissiles, only standard versions.

Reply:
Per IO, it can fire peacemaker and santa anna nukes, which is why those are in.